server/events.go

```diff
@@ -821,6 +825,10 @@ func (s *Server) updateServerUsage(v *ServerStats) {
 	var vss int64
 	pse.ProcUsage(&v.CPU, &v.Mem, &vss)
 	v.Cores = runtime.NumCPU()
+	v.MaxProcs = runtime.GOMAXPROCS(-1)
+	if mm := debug.SetMemoryLimit(-1); mm < math.MaxInt64 {
```

Comment:
Does this actually set a limit or just badly named get operation?

Reply:
Like `GOMAXPROCS`, passing a negative is a no-op. From https://cs.opensource.google/go/go/+/go1.24.2:src/runtime/debug/garbage.go;l=234:
> SetMemoryLimit returns the previously set memory limit. A negative input does not adjust the limit, and allows for retrieval of the currently set memory limit.